featurebyte/tile/manager.py

```diff
@@ -96,7 +95,7 @@ async def generate_tiles_on_demand(
 
             if progress_callback:
                 progress_callback(
-                    int(np.floor((index + 1) / num_jobs * TILE_COMPUTE_PROGRESS_MAX_PERCENT)),
+                    int(100 * np.floor((index + 1) / num_jobs)),
```

Comment:
Subtasks like this can focus on reporting the local progress from 0% to 100% without worrying about the global progress percentage that it occupies. That is determined by the caller that passes in the progress callback.